packages/components/src/components/Image/images.ts

```diff
@@ -58,6 +58,10 @@ export const PNG_IMAGES = {
     COINS_2x: 'coins@2x.png',
     TRADING_DCA_INVITY_APP_QR: 'trading-dca-invity-app-qr.png',
     TRADING_DCA_INVITY_APP_QR_2x: 'trading-dca-invity-app-qr@2x.png',
+    TRADING_DCA_INVITY_APP: 'trading-dca-invity-app.webp',
```

Comment:
image width is 508px so please set it as max-width for the image so that it is not bigger on smaller screens

<img width="1075" alt="Screenshot 2025-03-31 at 8 39 36" src="https://github.com/user-attachments/assets/2211e465-259d-4f15-9603-554835fe71e7" />